### ISSUE TITLE
Not working.

### ISSUE THREAD

**Issue by Nirvana2211** [Frackalyzer/OGWellDataSources#1]
It seems most of the links are broken. Do you intend to fix it sometime?

**Comment by Frackalyzer** [comment]
Yes, I do intend to fix the broken links, hopefully sooner than later. But
in the meantime, I have to feed my family whilst developing databases,
Python scripts, and bespoke websites on extremely short notices.

Not whining, but being the lone ranger is hard and I've not had anyone
volunteer to document a state's oil-and-gas links, not one, though I've had
one point me in the right direction once.

Tell you what, you document a state's links that need updating and I'll
post them when verified and cite you as the contributor. I'm not being
facetious nor snarky, but working as part of a team is better than doing so
alone.

Khepry Quixote
Software/Database Designer/Developer
Arkansas, USA

**Comment by Nirvana2211** [comment]
Hey Khepry - thanks for the reply.  I am a lone ranger as well, so I get you. I'll see what I can get for states and share the info with you.
